setup.py

```diff
@@ -37,6 +36,26 @@ def rglob(path, glob):
     "pyomo @ https://github.com/IDAES/pyomo/archive/6.3.0.zip"
 ]
 
+# For included DMF data
+DMF_DATA_ROOT = "data"
+
+
+def dmf_data_files(root: str = DMF_DATA_ROOT) -> List[Tuple[str, List[str]]]:
+    """Generate a list of pairs (directory, [files..]), covering all the DMF data
+       files, for the `data_files` option to :func:`setup()`.
+    """
+    file_list = [
+        (
+            root,
+            [f"{root}/config.yaml", f"{root}/resourcedb.json"],
+        )
+    ]
+    files_root = Path(root) / "files"
+    for files_subdir in files_root.glob("*"):
```

Comment:
I'm not sure how "dangerous" is this, but this seems to "trust" the fact that `files_subdir` (i.e. path directly under `files_root`) is a directory without checking/filtering, e.g. I think `files_subdir.glob()` would fail if it happens to be a file.